src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -171,10 +220,14 @@ export function useSwapInputsController({
       data,
```

Comment:
I liked Christian's idea of appending `Worklet` to worklet functions - leaving a note here for `setQuote`